docker/Dockerfile.nginx

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:20.04 AS base
```

Comment:
We need to build NGINX from source to include support for aio files and multiple architectures. `nginx-vod-module` has a repo we could add, but it is only built for `amd64`.